### DISCUSSION TITLE
WARNING Delta TMP_G. Mesh: 5 Particle: 3245 For Freezer Models

### DISCUSSION THREAD

**Discussion by norm07** [firemodels/fds#13401] [General]
Hi All,

I'm trying to model a fire in a freezer, and for some reason I'm having issues getting my sprinklers to work properly when I set the ambient temperature conditions to -20 degrees. I've attached my FDS file below, can someone please let me know what I'm missing with regards to how to set up sprinklers correctly? 
[222150_DF2_SEN.txt](https://github.com/user-attachments/files/16904047/222150_DF2_SEN.txt)

**Comment by drjfloyd** [comment]
You have set the initial temperature for sprinklers emitting pure water to be 20 degrees below freezing.

**Reply by norm07** [reply]
are the sprinkler heads not at -20 degrees when they are in a freezer room? Or can this reality not be captured in FDS?

**Reply by drjfloyd** [reply]
You are using WATER VAPOR as your liquid. Water freezes at 0 C. The default temperature for PART is TMPA. So you have told FDS that you are injecting ice cubes into the domain. If water supply feeding the sprinkler heads was -20 C your pipes would be frozen. If you actually have a wet pipe system then you would need some kind of anti freeze in any branch lines or rises inside the freeze and would no longer have pure water. At least for the first few moments until that mxiture was flushed out.

**Reply by norm07** [reply]
thank you, that makes a lot of sense, so my simulation ambient temperature can remain at the -20 degrees, but i should change my sprinkler initial temperature to 0 in order to prevent this error.

**Reply by norm07** [reply]
Ok i just changed the initial temperature of the sprinkler head back to 20 degrees and i still receive the same error :/

**Reply by drjfloyd** [reply]
The PROP temperature is the temperature of the sprinkler link. PART has its own input for the temperature of the fluid. The PROP temperature is not the issue. It is the PART temperature which you have not set so it is the ambient. I would set this to whatever the actual water supply temperature is which is not going to be 0 C since that would mean frozen pipes in your connection to the municipal supply.

**Reply by norm07** [reply]
Hu! it worked! I made the following adjustment:

&PART ID='Water01',
      SPEC_ID='WATER VAPOR',
      DIAMETER=746.0,
      AGE=60.0,
      SAMPLING_FACTOR=1,
      INITIAL_TEMPERATURE=20.0/
